plugins/@grouparoo/app-templates/src/source/query/records.ts

```diff
@@ -30,6 +31,7 @@ export const getRecordsMethod = (getChangedRows: GetChangedRowsMethod) => {
     appOptions,
     properties,
     sourceOptions,
+    schedule,
   }) => {
     let offset = highWaterMark.offset
```

Comment:
Is this also a case where a source is using highWatermark as a `sourceOffset`?

Reply:
It sure is! Good catch!